content/20.treatments.md

```diff
@@ -36,7 +36,7 @@ A list of current studies and their results, using carefully the information req
 
 Summarize the state of the symptom management approach.
 
-### Antivirals
+### Small Molecule Drugs for COVID-19
```

Comment:
```suggestion
### Small-Molecule Drugs for COVID-19
```

This is a trivial suggestion I'm making to capture the pictures I need for the tutorial, I hope you don't mind having your PR as the model!